src/common/components/game/PlayerArea.js

```diff
@@ -85,7 +85,7 @@ export default class PlayerArea extends Component {
         <PlayerName
           opponent={opponent}
           color={color}
-          playerName={(gameProps.player === color && !gameProps.isSandbox) ? 'You' : gameProps.usernames[color]} />
+          playerName={gameProps.usernames[color]} />
```

Comment:
Is the `isSandbox` case properly handled now?

Reply:
Yeah, `usernames` should always be passed in now.